.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ exclude: >
     bzl/version.tpl.py|
     ci/pipeline.yml|
     plaidbench/plaidbench/networks/keras/imdb_lstm.h5|
-    tile/targets/cpu/heatmap_skx_xeonplat_8180_1-7GHz_mblocked.csv.gz
+    pmlc/dialect/pxa/transforms/heatmap_skx_xeonplat_8180_1-7GHz_mblocked.csv.gz
```

Comment:
Let's put this in `pmlc/target/x86`, since it's only useful for XSMM/x86 targets

Reply:
Moved.